crypto/p256sm2_test.go

```diff
@@ -49,6 +50,12 @@ func Test256sm2(t *testing.T) {
 	pk1, err = ReadPublicKeyFromPem("pk.pem", pwd)
 	require.NoError(err)
 	require.Equal(pk, pk1)
+	require.NoError(UpdatePrivateKeyPasswordToPem("sk.pem", pwd, pwd2))
+	_, err = ReadPrivateKeyFromPem("sk.pem", pwd)
+	require.Error(err)
+	sk2, err = ReadPrivateKeyFromPem("sk.pem", pwd2)
+	require.NoError(err)
+	require.Equal(sk, sk2)
```

Comment:
can you add a case with new password = "", and read back
to cover the if else at line 83 above